src/ngraph/runtime/interpreter/int_executable.hpp

```diff
@@ -740,6 +740,29 @@ class ngraph::runtime::interpreter::INTExecutable : public Executable
                 args[0]->get_data_ptr<const T>(), out[0]->get_data_ptr<T>(), element_count);
             break;
         }
+        case OP_TYPEID::FunctionCall:
```

Comment:
Sorry, I do not understand a purpose of that operation.
Is it an attempt to emulate "graph level inlining"?
Do we have any performance statistics about diffrent approaches (generate two large graphs with similar part and the same with "FunctionCall")?
In particular, INTELGPU backend has no ability for native support this. Any FunctionCall will be "unrolled", or executed as independent graph with all required overhead (load to runtime stack and memory copy of the input parameters).

Reply:
FunctionCall is a collection of ops which are not supported by the current running backend. An example is integer ops with the NNP transformer. Integer ops must fall back to run on the CPU in that case. The FunctionCall is specifically because backends like the INTELGPU have no native support for some things, all op that are not supported natively are wrapped up in the FunctionCall and then called on an appropriate backend which does support those ops. The intent is to execute this "as independent graph with all required overhead (load to runtime stack and memory copy of the input parameters)". This is the hybrid transformer.
Currently two other approaches have been tried. One is "graph splitting" which ends up copying too much data back and forth between backends, and the "one op at a time" approach of the NNP transformer. Both of those have performance issues that are addressed with this PR.